agent/lib/kontena/network_adapters/weave.rb

```diff
@@ -81,12 +81,6 @@ def adapter_image?(image)
       false
     end
 
-    def router_image?(image)
```

Comment:
Moved this to the `WeaveWorker`, because:

* It was the only caller for it 
* It's now called for every `container:event`